src/System.Console/src/System/Console.cs

```diff
@@ -365,5 +366,61 @@ public static void Write(String value)
         {
             Out.Write(value);
         }
+
+        #region ctrlc
```

Comment:
I feel like this could have a more descriptive name (or just remove the `region`).